pkg/controller/jobs/pod/pod_controller_test.go

```diff
@@ -658,6 +660,7 @@ func TestReconciler(t *testing.T) {
 					Queue("user-queue").
 					ReserveQuota(utiltesting.MakeAdmission("cq").AssignmentPodCount(1).Obj()).
 					Admitted(true).
+					ReclaimablePods(kueue.ReclaimablePod{Name: "b990493b", Count: 1}).
```

Comment:
how come this works, but for the other test you need to skip?

Reply:
As far as I remember, the problem is in the creator of the workload. In this case, wl is created by the test, in another it's created by the reconciler. I will take a closer look.